app/src/main/res/values/strings.xml

```diff
@@ -95,6 +95,28 @@
     <string name="pref_play_vibration_after_rest_timer">Play vibration</string>
     <string name="pref_play_vibration_after_rest_timer_summary">Play vibration after rest timer finished</string>
 
+    <string name="pref_scheduled_backup_setting">Scheduled Backup Settings</string>
+    <string-array name="backup_options">
+        <item>Manually</item>
+        <item>Daily</item>
+        <item>Weekly</item>
+        <item>Monthly</item>
+    </string-array>
+
+    <string-array name="backup_settings_Enum" translatable="false">
+        <item>0</item>
+        <item>1</item>
+        <item>2</item>
+        <item>3</item>
+    </string-array>
+
+    <string name="pref_preferredBackupSettingSummary">Preferred Backup Setting:</string>
+    <string name="pref_lastBackupSettingSummary">Last Backup:</string>
+    <string name="pref_preferredBackupSettingTitle">Select your preferred Backup Setting</string>
```

Comment:
Just "Frequency" would be enough.

Reply:
Solved by 8d7006ef351c58d01ce4e23dcfeb8413e5987de0.